src/property.html

```diff
@@ -0,0 +1,1066 @@
+                  Location
+                </div>
+              </div>
+              <div class="p-2 bg-for-logo-section rounded-full">
+                <img
+                  src="/public/assets/propertypage/arrow-bottom.svg"
+                  alt="bottom-arrow-icon"
+                  class="h-4 w-4"
+                />
+              </div>
+            </div>
+            <!-- second -->
+            <div
+              class="flex justify-between gap-5 2xl:gap-12 items-center rounded-lg py-3 px-3.5 2xl:p-5 border-button text-color bg-color-all"
+            >
+              <div class="grid grid-flow-col items-center gap-2">
+                <div>
+                  <img
+                    src="/public/assets/propertypage/property.svg"
+                    alt="property"
+                  />
+                </div>
+                <div class="border-left h-5"></div>
+                <div class="text-color-addition-info text-sm 2xl:text-lg">
+                  Property Type
+                </div>
+              </div>
+              <div class="p-2 bg-for-logo-section rounded-full">
+                <img
+                  src="/public/assets/propertypage/arrow-bottom.svg"
+                  alt="bottom-arrow-icon"
+                  class="h-4 w-4"
+                />
+              </div>
+            </div>
+            <!-- third -->
+            <div
+              class="flex justify-between gap-5 2xl:gap-12 items-center rounded-lg py-3 px-3.5 2xl:p-5 border-button text-color bg-color-all"
+            >
+              <div class="grid grid-flow-col items-center gap-2">
+                <div>
+                  <img
+                    src="/public/assets/propertypage/range.svg"
+                    alt="range"
+                  />
+                </div>
+                <div class="border-left h-5"></div>
+                <div class="text-color-addition-info text-sm 2xl:text-lg">
+                  Pricing Range
+                </div>
+              </div>
+              <div class="p-2 bg-for-logo-section rounded-full">
+                <img
+                  src="/public/assets/propertypage/arrow-bottom.svg"
+                  alt="bottom-arrow-icon"
+                  class="h-4 w-4"
+                />
+              </div>
+            </div>
+            <!-- fourth -->
+            <div
+              class="flex justify-between gap-5 2xl:gap-12 items-center rounded-lg py-3 px-3.5 2xl:p-5 border-button text-color bg-color-all"
+            >
+              <div class="grid grid-flow-col items-center gap-2">
+                <div>
+                  <img src="/public/assets/propertypage/size.svg" alt="size" />
+                </div>
+                <div class="border-left h-5"></div>
+                <div class="text-color-addition-info text-sm 2xl:text-lg">
+                  Property Size
+                </div>
+              </div>
+              <div class="p-2 bg-for-logo-section rounded-full">
```

Comment:
shitty name bg-logo-for-section? Please make your you are in your senses while working, you are using shitty class names everywhere, please try to improve

Reply:
fixed it